settings.gradle

```diff
@@ -22,6 +22,8 @@ pluginManagement {
 include ":kotlin-build-common",
         ":compiler",
         ":compiler:util",
+        ":kotlin-util-io",
+        ":kotlin-util-klib",
```

Comment:
I'd prefer if these projects were named `:compiler:util-io` and `:compiler:util-klib` which is more similar to `:compiler:util` and reflects directory structure, which means less guesswork needed when trying to do anything with our build